### PR TITLE
Fixing duplicate menu entries

### PR DESCRIPTION
**Pull request by amvanbaren**
## Contains PRs:
- https://github.com/EclipseFdn/open-vsx.org/pull/1461

### CODE REVIEW

File: website/src/menu-content.tsx [hunk 1]
```diff
@@ -8,169 +8,159 @@
  * SPDX-License-Identifier: EPL-2.0
  ********************************************************************************/
 
-import * as React from 'react';
```
Comment (chrisguindon) [inline website/src/menu-content.tsx:11]: Looks like this patch is adding an extra space at the beginning of each line which makes it impossible for us to review the diff for this patch.

**Comment by chrisguindon** [comment]
I also notice that the ECA is failling with the last commit from @amvanbaren.
https://accounts.eclipse.org/legal/eca/validation/130934

@amvanbaren You must ensure that your contributions are made with your Eclipse account email address.  You are using an @hotmail.com email which is not tied to your work eclipse account.

File: website/src/menu-content.tsx (hunk 1 above)
Comment (amvanbaren) [reply website/src/menu-content.tsx]: I've removed the leading spaces, so that you can see the changes.

**Review by kineticsquid — APPROVED** [review]
Thanks @amvanbaren, looks good

**Pull request merged**